examples/pages/basic/index.js

```diff
@@ -108,23 +108,30 @@ function BasicAutocomplete({items, onChange}) {
   return (
     <Autocomplete onChange={onChange}>
       <Input placeholder="Favorite color ?" />
-      <Autocomplete.Menu style={{border: '1px solid rgba(34,36,38,.15)'}}>
-        {({inputValue, selectedItem, highlightedIndex}) =>
-          // prettier is doing weeeeird things to this
-          // prettier-ignore
-          items.filter(i => !inputValue || i.toLowerCase().includes(inputValue.toLowerCase()))
-            .map((item, index) => (
-              <Item
-                value={item}
-                index={index}
-                key={item}
-                highlightedIndex={highlightedIndex}
-                selectedItem={selectedItem}
-              >
-                {item}
-              </Item>
-            ))}
-      </Autocomplete.Menu>
+      <Autocomplete.Menu
+        style={{border: '1px solid rgba(34,36,38,.15)'}}
+        render={({isOpen, inputValue, selectedItem, highlightedIndex}) =>
+          isOpen &&
```

Comment:
I kinda like that the autocomplete component renders `null` for me if `isOpen` is true, that way I can avoid this check. Otherwise _everyone_ will need to do this... Unless you can think of another use case where I'd want to render if if `isOpen` is false... 🤔